python/paddle/tensor/math.py

```diff
@@ -7871,3 +7871,176 @@ def isreal(x, name=None):
         return paddle.ones_like(x, dtype='bool')
 
     return paddle.equal(paddle.imag(x), 0)
+
+
+def isin(elements, test_elements, assume_unique=False, invert=False, name=None):
```

Comment:
Accroding to [API design guidelines standard](https://www.paddlepaddle.org.cn/documentation/docs/zh/dev_guides/api_contributing_guides/api_design_guidelines_standard_cn.html), use `x` instead of `elements` and `test_x` instead of `test_elements`